features/support/world.js

```diff
@@ -100,6 +100,7 @@ function CustomWorld({ attach, parameters }) {
     this.instance = `${this.protocol}://${process.env.SD_API}`;
     this.testOrg = process.env.TEST_ORG;
     this.username = process.env.TEST_USERNAME;
+    this.scmContext = process.env.TEST_SCM_CONTEXT;
```

Comment:
Yes, If screwdriver.cd will be enable this feature in the future, we should set the setting.